### PR TITLE
Fix bug related to having multiple words as ingredient name

### PR DESCRIPTION
**Pull request by yowang20**

### CODE REVIEW

**Comment by ericjest** [comment]
@yowang20 do you still want us to review this/do you still want to merge this into main

**Comment by yowang20** [comment]
> @yowang20 do you still want us to review this/do you still want to merge this into main

yes please! wait maybe this is why TA couldn't do the green onion T_T I forgot this was unmerged.

**Comment by ericjest** [comment]
> > @yowang20 do you still want us to review this/do you still want to merge this into main
> 
> yes please! wait maybe this is why TA couldn't do the green onion T_T I forgot this was unmerged.

OK, I will get to it later tonight if I have time, but probably won't be able to until tomorrow. In the future, please keep track of your PRs and message people if they aren't getting reviewed.

File: src/main/java/controllers/PostController.java [hunk 1]
```diff
@@ -115,7 +115,13 @@ private String getMeasurableIngredients(String promptMeasurable) {
             } else if (strippedIngredient.split(" ").length < 3) {
                 inOut.setOutput(errorMessage);
                 measurableInput = getMeasurableIngredients(promptMeasurable);
-            }
+            } /*
```
Comment (sebastianibam002) [inline src/main/java/controllers/PostController.java:118]: Why did you add a comment. You could delete this part right?

**Review by sebastianibam002 — APPROVED** [review]

File: src/main/java/controllers/PostController.java (hunk 1 above)
Comment (yowang20) [reply src/main/java/controllers/PostController.java:118]: the comment continues after that line; it's a comment regarding another bug I found which I don' know how to solve yet

**Review by notnine — APPROVED** [review]
